parlai/scripts/build_dict.py

```diff
@@ -52,7 +52,7 @@ def setup_args(parser=None, hidden=True):
         hidden=hidden,
     )
     dict_loop.add_argument(
-        '-ltim', '--log-every-n-secs', type=float, default=2, hidden=hidden
+        '-ltim', '--log-every-n-secs', type=float, default=10, hidden=hidden
```

Comment:
is there any trick for setting those magic number: 2, 10, 20, etc?

Reply:
i think we usually pick these based on what feels right, no magic tricks 😄

Reply:
Yes, this one is just a personal preference. 2 seconds feels like logging too often, so I raised it :D